ChatApplication/Startup.cs

```diff
@@ -48,9 +50,9 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             services.AddSingleton<ICacheExtensionsService, CacheExtensionsService>();
+            services.AddTransient<IMessageService, MessageService>();
```

Comment:
Why did you inject IMessageService as a Transient what is the difference compared with Singleton?

Reply:
AddTransient 
A new service instance is created for each object in the Http request.
This is a approach for the multithreading approach because both objects are independent of one another.
Singleton
Only one service instance was created throughout the lifetime.
Reused the same instance in future, wherever the service is required.